test/unit/spec/nbextensions/appCell2/widgets/appCellWidget-spec.js

```diff
@@ -921,18 +1016,15 @@ define([
                         const busEmissions = this.bus.emit.calls.allArgs().filter((call) => {
                             return call[0] !== 'clock-tick';
                         });
-                        // the cell will emit at least one request for status updates when
-                        // the cell starts up
-                        expect(busEmissions).toContain([
-                            jcm.MESSAGE_TYPE.START_UPDATE,
-                            { [jcm.PARAM.JOB_ID]: jobId },
-                        ]);
-                        // if the job state is terminal, expect there to be a request to stop updates
-                        if (Jobs.isTerminalStatus(jobState.status)) {
+
+                        // if the job state is not terminal, expect there to be a job status request
+                        if (!Jobs.isTerminalStatus(jobState.status)) {
                             expect(busEmissions).toContain([
-                                jcm.MESSAGE_TYPE.STOP_UPDATE,
+                                jcm.MESSAGE_TYPE.STATUS,
                                 { [jcm.PARAM.JOB_ID]: jobId },
                             ]);
+                        } else {
+                            // expect(busEmissions.)
```

Comment:
I expect bus emissions to be rather stinky and diesel smelling. Perhaps natural gas smells?

Reply:
you definitely don't want to inhale too many of them!